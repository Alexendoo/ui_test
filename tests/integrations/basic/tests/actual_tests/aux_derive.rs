//@aux-build:derive_proc_macro.rs:proc-macro

#[macro_use]
extern crate derive_proc_macro;

fn main() {
    let x = Foo;
    x.bar();
}

#[derive(Something)]
struct Foo;
//~^ ERROR: no method
