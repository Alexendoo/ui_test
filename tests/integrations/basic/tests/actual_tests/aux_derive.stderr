error[E0599]: no method named `bar` found for struct `Foo` in the current scope
  --> $DIR/aux_derive.rs:8:7
   |
8  |     x.bar();
   |       ^^^ method not found in `Foo`
...
12 | struct Foo;
   | ---------- method `bar` not found for this struct

error: aborting due to previous error

For more information about this error, try `rustc --explain E0599`.
